### ISSUE TITLE
board disconnection/connection.

### ISSUE THREAD

**Issue by criticalboot** [rwaldron/etherport#10]
I am experiencing a problem of J5 server crash stop upon arduino disconnected from the network (ECONNRESET), please let me know how to handle this situation.

My understandings are below. Please correct if I am wrong.
J5 don't have board close/reconnect option - so we can't reconnect to arduino once it is detached from the network. unless we restart the server?

**Comment by soundanalogous** [comment]
That is correct. This functionality has not yet been implemented. There is work on the Firmata side to help out in determining the connection state on both the host and firmware side.
https://github.com/firmata/protocol/issues/4#issuecomment-214887160

**Comment by daftfox** [comment]
What is the status of this issue?
